front/components/sparkle/AppLayout.tsx

```diff
@@ -170,6 +171,7 @@ export default function AppLayout({
   navChildren?: React.ReactNode;
   titleChildren?: React.ReactNode;
   children: React.ReactNode;
+  hideSidebar?: boolean;
```

Comment:
let's mege this with isWideMode to `mode = "wide" | "full" | "normal"`

Reply:
as per IRL, keeping them separate as you could want full + wide or full + not wide (and wide + not full)